### PR TITLE
docs: user officer guide structure

### PR DESCRIPTION
**Pull request by sylvanab**
* Added new pages and template folder to User Officer folder
* Added images to the guide
* Created links to specific 'how-to' guides in the User Officer page

## Tests included/Docs Updated?

- [ ] I have added tests to cover my changes.
- [ ] All relevant doc has been updated

### CODE REVIEW

**Review by yoganandaness — APPROVED** [review]

**Comment by Junjiequan** [comment]
PR title must follow the structure of `type: title`. Otherwise the check will fail 
The type can only be one of the following `build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test|BREAKING CHANGE`

File: documentation/docs/user-guide/user-officer/user-officer.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# User Officer
+
+User officer
+
+Welcome to the User Office documentation for **User Officers!**
+
+
+![workflow](images/call.png){ width="300" } ![workflow](images/workflow.png){ width="300" }
```
Comment (janosbabik) [inline documentation/docs/user-guide/user-officer/user-officer.md:8]: The path should be `/assets/images/...`

File: documentation/docs/user-guide/user-officer/proposal.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Proposal
+
+
+## Edit a proposal 
+
+# How do I edit a proposal?
+
+**1. Find the proposal you wish to edit**
```
Comment (janosbabik) [inline documentation/docs/user-guide/user-officer/proposal.md:8]: The generated list is not shown correctly when there is no blank line at the beginning in some cases. Could you add a new line wherever it's needed?

Screenshot 2024-07-25 at 10 14 00
Screenshot 2024-07-25 at 10 14 09

File: documentation/docs/user-guide/user-officer/user-officer.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+* #### [How do I use features?](features.md)
+* #### [How do I use App settings?](app_settings)
+* #### [How do I edit a PDF](API.md)
+ 
+
+# Glossary
+
+* [What are templates?](templates_info)
+* [What are workflows?](workflows_info)
+* [What are proposal templates?](2proposal_template.md)
```
Comment (janosbabik) [inline documentation/docs/user-guide/user-officer/user-officer.md:103]: Some links are not updated, I guess: in this case, `templates/proposal_template.md`.

**Review by janosbabik — CHANGES_REQUESTED** [review]
This is a really nice job, Sylvana! Just a few small things I noticed that need to be updated.

File: documentation/docs/user-guide/user-officer/user-officer.md (hunk 1 above)
Comment (sylvanab) [reply documentation/docs/user-guide/user-officer/user-officer.md]: Thanks for the helpful suggestions Janos!

**Review by janosbabik — APPROVED** [review]

**Pull request merged**